plugins/com.google.cloud.tools.eclipse.appengine.standard.java8.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,14 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Google Cloud Tools for App Engine Standard Java 8 Tests
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.standard.java8.test
+Fragment-Host: com.google.cloud.tools.eclipse.appengine.standard.java8
+Bundle-Version: 0.0.1.qualifier
+Bundle-Vendor: Google, Inc.
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Import-Package: com.google.cloud.tools.appengine;version="0.2.11",
```

Comment:
Opened https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/384 but it seems the package versions are fixed in 0.3.2.  Let's fix this when we next update appengine-plugins-core.